### PR TITLE
[feat]: Two Type  Contact, Fax and  voicemail Case

### PR DESCRIPTION
**Pull request by SushilMallRC**
This PR contains features:
1. Two type contact, Customer and Contact.
2. Contains Fax and VoiceMail case.
3. Also contains user details from realtime instead of static.

### CODE REVIEW

File: src/adapters/netsuite/index.js [hunk 1]
```diff
@@ -85,11 +86,32 @@ async function findContact({ user, authHeader, phoneNumber, overridingFormat })
                     })
                 }
             }
+            //For Customer search
+            const customerInfo = await axios.post(
+                `https://${user.hostname.split(".")[0]}.suitetalk.api.netsuite.com/services/rest/query/v1/suiteql`,
+                {
+                    q: `SELECT id,firstname,middlename,lastname FROM customer WHERE phone = ${numberToQuery}`
```
Comment (DaKingKong) [inline src/adapters/netsuite/index.js:93]: quick check. Does netsuite just have one type of phone? Some CRMs have phone, mobile, workPhone etc.

File: src/adapters/netsuite/index.js [hunk 2]
```diff
@@ -62,7 +62,8 @@ async function findContact({ user, authHeader, phoneNumber, overridingFormat })
     const foundContacts = [];
```
Comment (DaKingKong) [inline src/adapters/netsuite/index.js:62]: Here we'll need the use of overridingFormat, to support non-E.164 formats like: (123) 456-7890. The way to do it is, the setup is in extension options which allows up to 3 formats in very simple matter - replacing all * signs one by one with raw number (if you get number +11234567890, then raw number is 1234567890, refer to Clio for further details.)

File: src/adapters/manifest.json [hunk 3]
```diff
@@ -300,6 +300,10 @@
             "contactPageUrl": "https://{hostname}/app/common/entity/contact.nl?id={contactId}",
```
Comment (DaKingKong) [inline src/adapters/manifest.json:300]: Does this work the same for Customer?

File: src/adapters/netsuite/index.js (hunk 2 above)
Comment (SushilMallRC) [reply src/adapters/netsuite/index.js:62]: Addressed

File: src/adapters/manifest.json (hunk 3 above)
Comment (SushilMallRC) [reply src/adapters/manifest.json:300]: address, change contactType dynamically

**Pull request merged**